### ISSUE TITLE
Console focus indicator is chopped off on the right edge when there is only one console session

### ISSUE THREAD

**Issue by softwarenerd** [posit-dev/positron#14176]
#### Positron and OS details:

Positron Version: 2026.07.0 build 97
Code - OSS Version: 1.118.0
Commit: 26e4f065fe3c04a798de671dfa3e2eee7a840a58
Date: 2026-06-08T19:17:24-04:00
Electron: 39.8.8
Chromium: 142.0.7444.265
Node.js: 22.22.1
V8: 14.2.231.22-electron.0
OS: Darwin arm64 25.5.0

#### Session details:

Doesn't matter

## Describe the issue:

The console focus indicator is chopped off on the right edge when there is only one console session.

One session:

<p>
<img width="780" height="494" alt="Image" src="https://github.com/user-attachments/assets/45a84fae-d362-4a22-acad-8a941d64f9c7" />
</p>

Multiple sessions:

<p>
<img width="778" height="492" alt="Image" src="https://github.com/user-attachments/assets/3c932ac7-1d1c-4e2f-8183-b5d9a4ef3c33" />
</p>

## Steps to reproduce the issue:

1. Open Console with one session.
2. Click on the CWD to bring the focus indicator into view.
3. Note that the right edge is chopped off.
4. Add a session.
5. Click on the CWD to bring the focus indicator into view.
6. Note that the right edge is now visible.

## Expected or desired behavior:

The focus indicator should have all four sides.

## Were there any error messages in the UI, Output panel, or Developer Tools console?

N/A

**Comment by dhruvisompura** [comment]
Note: this problem shows up specifically after you create multiple sessions and then delete them until you have one. That's when the focus ring looks to be cut off on the right side.

**Comment by jmcphers** [comment]
This is true of a _bunch_ of our panes. https://github.com/posit-dev/positron/issues/8929

**Comment by jmcphers** [comment]
Triage: We can track this in https://github.com/posit-dev/positron/issues/8929.
